nautobot_device_lifecycle_mgmt/templates/nautobot_device_lifecycle_mgmt/softwarelcm.html

```diff
@@ -1,130 +1,73 @@
-{% extends 'base.html' %}
-{% load plugins %}
-{% load buttons %}
-{% load custom_links %}
-{% load helpers %}
+{% extends 'generic/object_detail.html' %}
 
-{% block title %}{{ object }}{% endblock %}
+{% block masthead %}
+    <h2>{% block title %}{{ object }}{% endblock %}</h2>
+{% endblock masthead %}
 
-{% block header %}
-    <div class="row noprint">
-        <div class="col-sm-8 col-md-9">
-            <ol class="breadcrumb">
-                <li><a href="{% url 'plugins:nautobot_device_lifecycle_mgmt:softwarelcm_list' %}">Software</a></li>
-                <li><a>{{ object }}</a></li>
-            </ol>
-        </div>
-        <div class="col-sm-4 col-md-3">
-            <form action="{% url 'plugins:nautobot_device_lifecycle_mgmt:softwarelcm_list' %}" method="get">
-                <div class="input-group">
-                    <input type="text" name="q" class="form-control" />
-                    <span class="input-group-btn">
-                        <button type="submit" class="btn btn-primary">
-                            <span class="mdi mdi-magnify" aria-hidden="true"></span> Search
-                        </button>
-                    </span>
-                </div>
-            </form>
-        </div>
-    </div>
-    <div class="pull-right noprint">
-        {% plugin_buttons object %}
-        {% if perms.nautobot_device_lifecycle_mgmt.add_softwareimagelcm and active_tab == 'software-images' %}
-            <a href="{% url 'plugins:nautobot_device_lifecycle_mgmt:softwareimagelcm_add' %}?software={{ object.pk }}" class="btn btn-success">
-                <span class="mdi mdi-plus-thick" aria-hidden="true"></span>
-                Add a Software Image
-            </a>
-        {% endif %}
```

Comment:
Can we replicate functionality provided by this code? 

Proposed changes remove the `Add a Software Image` button.

Before:

![image](https://user-images.githubusercontent.com/27512690/229898570-4beeb134-691a-4329-894b-14e84f1ee70a.png)


After:

![image](https://user-images.githubusercontent.com/27512690/229898477-aa827c59-d3b5-4e74-a196-5c8b9b99c955.png)

Reply:
Fixed, thanks for catching this @progala

Reply:
Thanks @snaselj , looking great!